actions/online-sign/action.yml

```diff
@@ -26,6 +34,13 @@ runs:
         workload_identity_provider: ${{ inputs.gcp_workload_identity_provider }}
         service_account: ${{ inputs.gcp_service_account }}
 
+    - name: Authenticate to AWS
+      if: inputs.aws_role_to_assume != ''
+      uses: aws-actions/configure-aws-credentials@v4
```

Comment:
could use the last release commit hash + version comment here like the other actions do